pkgs/development/tools/analysis/kcov/default.nix

```diff
@@ -10,34 +10,23 @@
 , python3
 , libiberty
 , libopcodes
-, runCommand
-, gcc
+, runCommandCC
 , rustc
 }:
 
 let
   self =
     stdenv.mkDerivation rec {
       pname = "kcov";
-      version = "38";
+      version = "41";
 
       src = fetchFromGitHub {
         owner = "SimonKagstrom";
         repo = "kcov";
         rev = "v${version}";
-        sha256 = "sha256-6LoIo2/yMUz8qIpwJVcA3qZjjF+8KEM1MyHuyHsQD38=";
+        sha256 = "sha256-Kit4Yn5Qeg3uAc6+RxwlVEhDKN6at+Uc7V38yhDPrAY=";
       };
 
-      patches = [
-        # Pull upstream patch for binutils-2/39 support:
-        #   https://github.com/SimonKagstrom/kcov/pull/383
-        (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Done